### PR TITLE
EZEE-2982: As an Editor, I want to edit content in the UDW

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZEE-2982
**Requires:** https://github.com/ezsystems/ezplatform-admin-ui/pull/1273

### CODE REVIEW

File: Resources/public/scss/modules/universal-discovery/_content.edit.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        height: calculateRem(50px);
+    }
+
+    &__cancel-button {
+        background-color: $ez-color-base-medium;
+        color: $ez-white;
+        margin-right: calculateRem(16px);
+
+        &:hover {
+            color: $ez-white;
```
Comment (GrabowskiM) [inline Resources/public/scss/modules/universal-discovery/_content.edit.scss:29]: Is this hover needed? It adds same property as in standard benahiour

File: src/modules/universal-discovery/components/content-edit-button/content.edit.button.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    const restInfo = useContext(RestInfoContext);
+    const allowRedirects = useContext(AllowRedirectsContext);
+    const [editOnTheFlyData, setEditOnTheFlyData] = useContext(EditOnTheFlyDataContext);
+    const [activeTab, setActiveTab] = useContext(ActiveTabContext);
+    const contentTypesMap = useContext(ContentTypesMapContext);
+    const [isTranslationSelectorVisible, setIsTranslationSelectorVisible] = useState(false);
+
+    useEffect(() => {
+        setIsTranslationSelectorVisible(false);
+    }, [version]);
```
Comment (OstafinL) [inline src/modules/universal-discovery/components/content-edit-button/content.edit.button.js:25]: Could you add empty line?

File: src/modules/universal-discovery/components/content-edit-button/content.edit.button.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    const hideTranslationSelector = () => {
```
Comment (GrabowskiM) [inline src/modules/universal-discovery/components/content-edit-button/content.edit.button.js:27]: New line to separate useEffect from declarations?

File: src/modules/universal-discovery/components/content-edit-button/content.edit.button.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        createDraft(
+            {
+                ...restInfo,
+                contentId,
+            },
+            (response) => redirectToContentEdit(contentId, response.Version.VersionInfo.versionNo, languageCode, location.id)
+        );
+    };
+    const renderTranslationSelector = () => {
+        return (
```
Comment (GrabowskiM) [inline src/modules/universal-discovery/components/content-edit-button/content.edit.button.js:80]: could be short return, but not that big issue

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

File: Resources/public/scss/modules/universal-discovery/_content.edit.scss (hunk 1 above)
Comment (dew326) [reply Resources/public/scss/modules/universal-discovery/_content.edit.scss:29]: It is. It overrides the default hover color

File: src/modules/universal-discovery/components/content-edit-button/content.edit.button.js (hunk 4 above)
Comment (dew326) [reply src/modules/universal-discovery/components/content-edit-button/content.edit.button.js:80]: I think it's more readable.

**Review by GrabowskiM — APPROVED** [review]

**Review by OstafinL — APPROVED** [review]

**Pull request merged**